RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -302,8 +302,8 @@
         RecoTracker.FinalTrackSelectors.multiTrackSelector_cfi.highpurityMTS.clone(
             name = 'highPtTripletStep',
             preFilterName = 'highPtTripletStepTight',
-            min_eta = -4.0,
-            max_eta = 4.0,
+#            min_eta = -4.0, # this steps the fake/eff is small for |eta|>4.0, and such a cut looks noticeably harmful
+#            max_eta = 4.0,
```

Comment:
ehm, I was actually implying that here we can use 4.1 
fr https://user-images.githubusercontent.com/4676718/61416295-0d24d300-a8a8-11e9-8fd1-66d230949be9.png
and eff https://user-images.githubusercontent.com/4676718/61416296-0d24d300-a8a8-11e9-8a3d-f4616b9fc166.png
or, better, a log-scale of eff https://tinyurl.com/y56bx7j9
![eff_h3HP_logy](https://user-images.githubusercontent.com/4676718/61662415-403ddc80-ac83-11e9-941a-c9bcde4a7721.png)

plots suggest that the balance of FR vs eff is bad for |eta|>4.1

Reply:
opsi, right, thanks @slava77 
I got confused

I've just updated the PR w/ the following

InitialStep_cff.py:                   min_eta = -4.1,
HighPtTripletStep_cff.py:      min_eta = -4.1
LowPtQuadStep_cff.py:         min_eta = -4.0,
LowPtTripletStep_cff.py:       min_eta = -4.0,
DetachedQuadStep_cff.py:   min_eta = -4.0,
PixelPairStep_cff.py:#            min_eta = -4.0, # useless, the eta distribution does not even reach |eta| = 3 (!)